fdbclient/Schemas.cpp

```diff
@@ -25,9 +25,12 @@ const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
 {
    "cluster":{
       "storage_wiggler": {
+         "error": "some error description",
          "wiggle_server_ids":["0ccb4e0feddb55"],
          "wiggle_server_addresses": ["127.0.0.1"],
          "primary": {
+            "state": {"$enum":["running", "paused", "unknown"]},
```

Comment:
Is there a reason to not have both `_timestamp` and `_datetime` for each time value?

Reply:
The datetime strings are useful because humans often look at the JSON document, but the epochs timestamps are the most important since this is meant to be easy to work with programmatically.  I think both would provide the most usability.